es.json

```diff
@@ -236,6 +248,11 @@
   "fantoche": 2,
   "fariseo": 2,
   "fiesta de salchichas": 2,
+  "fiesta de pollas": 2,
+  "fiesta de putas": 2,
+  "fiesta de culos": 2,
+  "fiesta de imbeciles": 2,
```

Comment:
It might be better to create a plural version for `imbecil` and put `imbeciles` on line 303. Putting words like this (like `putas`) will also cause double errors to be caught in the parser too should this phrase arise.